src/redux/configStore.js

```diff
@@ -0,0 +1,11 @@
+import { createStore, combineReducers, applyMiddleware } from 'redux';
+import logger from 'redux-logger';
+import booksReducer from './books/books';
+
+const configReducer = combineReducers({
+  booksReducer,
+});
```

Comment:
- [ ] After setting up the categories-related redux actions and reducer, ensure you import the reducer here, just like `booksReducer`.

- [ ] Add the imported categories reducer to the `combineReducers`.